tests/commands/test_store_charmlibs.py

```diff
@@ -0,0 +1,331 @@
+# Copyright 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For further info, check https://github.com/canonical/charmcraft
+
+"""Tests for store helpers commands (code in store/charmlibs.py)."""
+import hashlib
+import pathlib
+import sys
+
+import pytest
+from pytest_check import check
+from craft_cli import CraftError
+
+from charmcraft.commands.store.charmlibs import get_positive_int, get_lib_info
+
+
+class TestGetPositiveInt:
+    """Tests for get_positive_int."""
+
+    @pytest.mark.parametrize(
+        ["raw_value", "expected"],
+        [
+            pytest.param("1", 1),
+            pytest.param(b"0", 0),
+        ],
+    )
+    def test_success(self, raw_value, expected):
+        assert get_positive_int(raw_value) == expected
+
+    @pytest.mark.parametrize(
+        ["raw_value", "error_class", "error_regex"],
+        [
+            pytest.param("123.456", ValueError, "invalid literal"),
+            pytest.param("-1", ValueError, "negative"),
+            pytest.param(b"", ValueError, "invalid literal", id="empty-bytes"),
+        ],
+    )
+    def test_exceptions(self, raw_value, error_class, error_regex):
+        """Test various exceptions that can be passed through."""
+        with pytest.raises(error_class, match=error_regex):
+            get_positive_int(raw_value)
+
+
+class TestGetLibInfo:
```

Comment:
We're trying to move away from class-structured tests in pytest. Is there specific functionality that you need here?